src/components/NavigationDesktop.tsx

```diff
@@ -33,59 +29,54 @@ function flattenNavData(navData: NavData): NavData {
   return ret
 }
 
-export const NavigationDesktop = styled(
-  ({ ...props }: ComponentProps<'div'>) => {
-    const navData = useNavData()
-    const flatNav = useMemo(() => flattenNavData(navData), [navData])
+export function NavigationDesktop({ logoRef }) {
+  const navData = useNavData()
+  const flatNav = useMemo(() => flattenNavData(navData), [navData])
+  const logoLeft = logoRef?.current?.getBoundingClientRect()?.left
 
-    return (
-      <ColorModeProvider mode="dark">
-        <div {...props}>
-          {flatNav?.map((navItem, i) => {
-            if (navItem?.mobile_only) {
-              return null
+  return (
+    <ColorModeProvider mode="dark">
+      <div className="hidden gap-xsmall lg:flex">
+        {flatNav?.map((navItem, i) => {
+          if (navItem?.mobile_only) {
+            return null
+          }
+          if (navItem?.subnav) {
+            if (!i) {
```

Comment:
can we check the ids directly instead of using indexes for these? might be hard to debug if someone changes the order around in the future